plugins/rebalance/utils.go

```diff
@@ -0,0 +1,123 @@
+package rebalance
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/gluster/glusterd2/glusterd2/store"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	rebalanceapi "github.com/gluster/glusterd2/plugins/rebalance/api"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	rebalancePrefix string = "rebalance/"
+)
+
+// TimeVal represents the time value
+type TimeVal struct {
+	TVsec  uint64
+	TVusec uint64
+}
+
+var (
+	hash uint64
+)
+
+func glusterdVolInfoResetStats(r *rebalanceapi.RebalInfo) {
```

Comment:
This function is not required now, add this while working on the Rebalance status(Node level)